ui/src/app/views/account-creation/component.html

```diff
@@ -14,11 +14,33 @@ <h2>Create your account</h2>
       </div>
     </div>
     <div class="form-section">
-      <input type="text" class="short-input" id="username" name="username" [(ngModel)]="profile.username" (ngModelChange)="usernameChanged()" placeholder="Create Account">
-      <h5 style="display: inline-block;">@{{gsuiteDomain}}</h5>
+      <input type="text" class="short-input" id="username" name="username" [(ngModel)]="profile.username" (ngModelChange)="usernameChanged()" placeholder="Username">
+      <h5 style="display: inline-block;" class="domain">@{{gsuiteDomain}}</h5>
+      <clr-tooltip>
+        <clr-icon clrTooltipTrigger shape="info-standard" class="is-solid info-icon"></clr-icon>
+        <clr-tooltip-content clrPosition="top-right" clrSize="lg" *clrIfOpen>
+          <div>
+            Usernames can contain letters (a-z), numbers (0-9), dashes (-), underscores (_),
+            apostrophies ('), and periods (.).
+          </div>
+          <br>
+          <div>
+            Usernames can't contain an ampersand (&), equal sign (=), brackets (<,>), plus sign (+),
```

Comment:
I realize this is an issue in the Google documentation as well.. but I assume the first bullet is exhaustive. The only new information given here is that we can't have more than one "." in a row.

Reply:
I'm going to send this PR link over to Lou and have him weigh in on this. (I am mostly just following the mocks, which I assume are following the google documentation)